File: server/src/main/kotlin/com/pidulgi/server/member/dto/response/MemberGetResponse.kt
package com.pidulgi.server.member.dto.response

import com.pidulgi.server.member.entity.type.Gender
import java.time.LocalDateTime

data class MemberGetResponse(

    val memberId: Long,
    val images: List<MemberImageResponse> = emptyList(),
    val nickname: String,
    val gender: Gender,
    val age: Int,
    val bio: String? = null,
    val likes: Long,
    val distance: Double? = null,
    val updatedAt: LocalDateTime,
    val isLiked: Boolean,
    val isBlocked: Boolean,
    val isPrivateImageGranted: Boolean,
)

File: app/app/Model/Member/Response/MemberGetResponse.swift
struct MemberGetResponse: Codable, Equatable {

    let memberId: Int64
    let images: [MemberImageResponse]
    let nickname: String
    let gender: String
    let age: Int
    let bio: String?
    var likes: Int64
    let distance: Double?
    let updatedAt: String
    let isChatEnabled: Bool
    var isLiked: Bool
    var isBlocked: Bool
    var isPrivateImageGranted: Bool
    let isPrivateImageGrantedByTarget: Bool
}
